x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/transform/TransformMessages.java

```diff
@@ -30,6 +30,8 @@
     public static final String UNKNOWN_TRANSFORM_STATS = "Statistics for transform [{0}] could not be found";
 
     public static final String REST_DEPRECATED_ENDPOINT = "[_data_frame/transforms/] is deprecated, use [_transform/] in the future.";
+    public static final String REST_WARN_NO_TRANSFORM_NODES =
+        "Transform requires the transform node role for at least 1 node, found [{0}] transform nodes";
```

Comment:
Is the `[{0}]` parameter ever going to be non-zero? By looking at the code it seems impossible so maybe lets hardcode `0` here?